app/controllers/orders_controller.rb

```diff
@@ -2,7 +2,11 @@ class OrdersController < ApplicationController
   before_action :authenticate_employee!
 
   def index
-    @orders = current_employee.orders
+    @orders = if params['delivery_status'].nil?
+                current_employee.orders
+              else
+                current_employee.orders.filter_by_delivery_status(params[:delivery_status])
+              end
```

Comment:
Maybe we can try to learn something new and move filtering into Query Object :)
[How to use Query Objects to refactor Rails SQL-queries](https://mkdev.me/posts/how-to-use-query-objects-to-refactor-rails-sql-queries)